src/NHSD.BuyingCatalogue.Ordering.Persistence/Data/ApplicationDbContext.cs

```diff
@@ -27,6 +27,7 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             base.OnModelCreating(modelBuilder);
 
             modelBuilder.ApplyConfiguration(new OrderConfiguration());
+            modelBuilder.Entity<ServiceRecipient>(x => x.HasKey(y => y.OdsCode));
```

Comment:
Could the ServiceRecipient configuration be encapsulated into an entity configuration class to be consistent with the `OrderConfiguration` class above?

Reply:
👍